[2022-05-18 23:01:26,571] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_great_expectations_dag.dbt_docs scheduled__2022-05-17T22:12:00.138285+00:00 [queued]>
[2022-05-18 23:01:26,589] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_great_expectations_dag.dbt_docs scheduled__2022-05-17T22:12:00.138285+00:00 [queued]>
[2022-05-18 23:01:26,590] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-18 23:01:26,591] {taskinstance.py:1242} INFO - Starting attempt 2 of 5
[2022-05-18 23:01:26,592] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-18 23:01:26,607] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): dbt_docs> on 2022-05-17 22:12:00.138285+00:00
[2022-05-18 23:01:26,617] {standard_task_runner.py:52} INFO - Started process 177 to run task
[2022-05-18 23:01:26,625] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'model_level_dbt_great_expectations_dag', 'dbt_docs', 'scheduled__2022-05-17T22:12:00.138285+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/model_level_dbt_great_expectations_dag.py', '--cfg-path', '/tmp/tmp8zmc73f1', '--error-file', '/tmp/tmpi7xhauu5']
[2022-05-18 23:01:26,626] {standard_task_runner.py:77} INFO - Job 129: Subtask dbt_docs
[2022-05-18 23:01:26,720] {logging_mixin.py:109} INFO - Running <TaskInstance: model_level_dbt_great_expectations_dag.dbt_docs scheduled__2022-05-17T22:12:00.138285+00:00 [running]> on host 3d7cb1bfa13d
[2022-05-18 23:01:26,941] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=luciano.naveiro
AIRFLOW_CTX_DAG_ID=model_level_dbt_great_expectations_dag
AIRFLOW_CTX_TASK_ID=dbt_docs
AIRFLOW_CTX_EXECUTION_DATE=2022-05-17T22:12:00.138285+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-17T22:12:00.138285+00:00
[2022-05-18 23:01:26,945] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-18 23:01:26,947] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n        dbt docs generate --profiles-dir /opt/dbt/jaffle_shop --project-dir /opt/dbt/jaffle_shop\n        ']
[2022-05-18 23:01:26,976] {subprocess.py:85} INFO - Output:
[2022-05-18 23:01:31,835] {subprocess.py:89} INFO - 23:01:31  Running with dbt=1.1.0
[2022-05-18 23:01:32,228] {subprocess.py:89} INFO - 23:01:32  Found 5 models, 13 tests, 0 snapshots, 0 analyses, 170 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2022-05-18 23:01:32,256] {subprocess.py:89} INFO - 23:01:32
[2022-05-18 23:01:32,409] {subprocess.py:89} INFO - 23:01:32  Concurrency: 1 threads (target='dev')
[2022-05-18 23:01:32,414] {subprocess.py:89} INFO - 23:01:32
[2022-05-18 23:01:33,385] {subprocess.py:89} INFO - 23:01:33  Done.
[2022-05-18 23:01:33,652] {subprocess.py:89} INFO - 23:01:33  Building catalog
[2022-05-18 23:01:33,776] {subprocess.py:89} INFO - 23:01:33  Catalog written to /opt/dbt/jaffle_shop/target/catalog.json
[2022-05-18 23:01:34,973] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-18 23:01:35,007] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=model_level_dbt_great_expectations_dag, task_id=dbt_docs, execution_date=20220517T221200, start_date=20220518T230126, end_date=20220518T230135
[2022-05-18 23:01:35,051] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-18 23:01:35,097] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
